src/components/navigation/Accordion.test.tsx

```diff
@@ -128,7 +128,9 @@ describe('Accordion', () => {
 		act(() => {
 			userEvent.click(screen.getByText('First'));
 		});
-		expect(onClick).toHaveBeenCalled();
+		act(() => {
+			userEvent.click(container.getElementsByTagName('svg')[0]);
+		});
```

Comment:
It seems to me that this change is made "to make the test works". What about improving the existing test, since the behaviour of the component is now changed?

Something like
```typescript
test('Open and close an Accordion', async () => {
		const onClick = jest.fn();
		render(
			<Accordion
				items={[
					{
						id: 'first',
						label: 'First',
						onClick,
						items: [
							{ id: 'second', label: 'Second' },
							{ id: 'third', label: 'Third' },
							{ id: 'fourth', label: 'Fourth' }
						]
					}
				]}
			/>
		);

		expect(screen.getByText(/first/i)).toBeVisible();
		expect(screen.getByText(/second/i)).not.toBeVisible();
		// click on label does not expand the accordion
		userEvent.click(screen.getByText('First'));
		expect(onClick).toHaveBeenCalledTimes(1);
		expect(screen.getByText(/second/i)).not.toBeVisible();
		// click on chevron icon expand the accordion item
		userEvent.click(screen.getByTestId('icon: ChevronDown'));
		await waitFor(() => expect(screen.getByText(/second/i)).toBeVisible());
		// click on chevron icon does not call onClick callback
		expect(onClick).toHaveBeenCalledTimes(1);
		expect(screen.getByText(/second/i)).toBeVisible();
		expect(screen.getByTestId('icon: ChevronUp')).toBeVisible();
		expect(screen.queryByTestId('icon: ChevronDown')).not.toBeInTheDocument();
		// click on chevron icon of opened accordion close the accordion and does not call onClick callback
		userEvent.click(screen.getByTestId('icon: ChevronUp'));
		await waitFor(() => expect(screen.getByText(/second/i)).not.toBeVisible());
		expect(onClick).toHaveBeenCalledTimes(1);
		expect(screen.getByTestId('icon: ChevronDown')).toBeVisible();
		expect(screen.queryByTestId('icon: ChevronUp')).not.toBeInTheDocument();
});
```

Reply:
Pushed updated version